etc/config/c++.amazon.properties

```diff
@@ -383,6 +383,16 @@ compiler.raspbian9.exe=/opt/compiler-explorer/arm/raspbian9-6.3.0/bin/arm-raspbi
 compiler.raspbian9.name=Raspbian Stretch
 compiler.raspbian9.semver=6.3
 compiler.raspbian9.objdumper=/opt/compiler-explorer/arm/raspbian9-6.3.0/bin/arm-raspbian9-linux-gnueabihf-objdump
+compiler.arduinouno.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-g++
```

Comment:
Can you add some of the version into this ID? The IDs need to remain stable forever, and I've found it best over the years to put the version (540 maybe?) or something germaine to this specific setup in the name. Thanks